index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>a Tribute Page</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <main id="main">
+    <h1 id="title">Dr.Norman Borlaug</h1>
+    <p id="subtext">The man who saved a billion lives</p>
+    <div id="img-div">
+      <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt=""
+        id="image">
+      <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
+        yields - part of his life-long war on hunger.</p>
+    </div>
```

Comment:
You should use HTML5 semantic tags (use `figure` to wrap the image and its caption and `figcaption` instead of `p` to write the figure caption). ✍️

Reply:
- Kindly try to use suggestions instead of orders like what you did in the first two comments.